src/scalarstats.jl

```diff
@@ -153,9 +181,21 @@ end
 #
 #############################
 
+"""
+    percentile(v, p) -> Float
+
+Return the `p`th percentile of an array `v`.
+"""
 percentile{T<:Real}(v::AbstractArray{T}, p) = quantile(v, p * 0.01)
 
 quantile{T<:Real}(v::AbstractArray{T}) = quantile(v, [.0, .25, .5, .75, 1.0])
+
+"""
+    nquantile(v, n) -> Array
+
+Return the n-quantiles of an array, i.e. the values which partition `v` into
```

Comment:
Could you say which of the quantile algorithms is used? I know there are many variants of them. (Should also be done for `quantile` and `wquantile`.)